package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "audion",
+  "version": "1.0.0",
+  "description": "Traces web audio API calls.",
+  "main": "index.js",
+  "engines": {
+    "node": "14"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.14.6",
```

Comment:
Why do we need Babel? I thought we'd use vanilla JS for the implementation.

Reply:
It is used in the `jest` configuration. `jest` doesn't support es module syntax out of the box. The babel configuration here uses one babel plugin `@babel/plugin-transform-modules-commonjs` to transform import/export statements to commonjs syntax.